lib/handler/status/events.c

```diff
@@ -97,13 +97,14 @@ static h2o_iovec_t events_status_final(void *priv, h2o_globalconf_t *gconf, h2o_
                        " \"http2-errors.inadequate-security\": %" PRIu64 ", \n"
                        " \"http2.read-closed\": %" PRIu64 ", \n"
                        " \"http2.write-closed\": %" PRIu64 ", \n"
-                       " \"ssl.errors\": %" PRIu64 "\n",
+                       " \"ssl.errors\": %" PRIu64 ", \n"
+                       " \"memory.mmap_errors\": %" PRIu64 "\n",
```

Comment:
Let's use `%zu` instead of casting to `uint64_t` and using `PRIu64`.

The alternative is to change the type of `mmap_errors` to `uint64_t`, though I'd prefer not doing that because some 32-bit platforms might not provide atomic operation support for `uint64_t`.